src/messageHandler.ts

```diff
@@ -0,0 +1,22 @@
+import { Client, Message } from "discord.js";
+const BOT_ID = "863633669989466142"
+// Refer to https://github.com/discordjs/discord.js/blob/master/typings/index.d.ts for DiscordJS typings
+
+export interface IMessageHandler {
+  client: Client;
+  receivedMessage:(message : Message) => void;
```

Comment:
do we need more handlers for failed messages?

Reply:
Nope